tutorial/write-custom-widget.md

```diff
@@ -283,15 +281,178 @@ If `myWidgetRenderer` does NOT implement the `fyne.WidgetRenderer` interface thi
 
 ```go
 func main() {
-	app := app.New()
-	w := app.NewWindow("My Widget")
-	mw := NewMyWidget("Widget")
-	w.SetContent(mw)
-	w.ShowAndRun()
+    app := app.New()
+    w := app.NewWindow("My Widget")
+    mw := NewMyWidget("Widget")
+    w.SetContent(mw)
+    w.ShowAndRun()
 }
 ```
+
 The widget should look like this:
-	
-![widgetDemoScreenSmall](https://user-images.githubusercontent.com/94919638/145826575-53ae12fc-e6d0-4976-9037-dbdaaea9f4de.png)
 
-	The widget can be resized and the text will remain centered.
+![](../images/widgets/customWidgetSimple.png)
+
+The widget can be resized and the text will remain centered.
+
+### The complete Widget
+
+Note that some minor additions have be made to the code below:
```

Comment:
Why add more code? The complete demo should be as minimal as the code above, otherwise you are adding more items that are not explained.
Additionally this code uses deprecated APIs - we should not encourage developers to override user theme settings.